tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, name, init_config, instances):
         self.timeout = float(instance.get('timeout', 10))
         self.collect_response_time = instance.get('collect_response_time', False)
         self.url = instance.get('host', None)
+        self.ip_cache_duration = int(instance.get('ip_cache_duration', 0)) or None
```

Comment:
```suggestion
        self.ip_cache_duration = int(instance.get('ip_cache_duration', 120)) or None
```

Reply:
Or better, set it as a constant

Reply:
I actually intended for a default value of `None` if unset instead of a constant value.
Reason why I defaulted to `0` is because `int(None) or None` raises an error but `int(0) or None` returns the default `None`